tests/Controller/LicenseControllerTest.php

```diff
@@ -28,6 +28,28 @@ protected function setUp(): void
         $this->commercialLicense = $this->createMock(CommercialLicense::class);
     }
 
+    public function testSyncWithValidLicenseHost(): void
```

Comment:
Maybe a test with both just for completeness? But i approve anyway, so merge when you want